x/upgrade/module.go

```diff
@@ -223,6 +226,10 @@ func ProvideModule(in ModuleInputs) ModuleOutputs {
 	m := NewAppModule(k, in.AddressCodec)
 	gh := govv1beta1.HandlerRoute{RouteKey: types.RouterKey, Handler: NewSoftwareUpgradeProposalHandler(k)}
 
+	if upgradePlan, err := k.ReadUpgradeInfoFromDisk(); err == nil {
```

Comment:
I have not tried if it works, but if it does it will only work for app wiring app (app_v2). Could you add this additionally to the app.go?